docs/how-to-guides/use-connector-management.md

```diff
@@ -0,0 +1,244 @@
+# How to use connector management
+
+## Context
+
+You have external data stores that you want to read from and write to with ksqlDB, but you don’t want to write custom glue code to do it. ksqlDB is capable of using the vast ecosystem of Kafka Connect connectors through its SQL syntax. This functionality is called "connector management".
+
+## In action
+
+```sql
+CREATE SOURCE CONNECTOR s WITH (
+  'connector.class' = 'io.mdrogalis.voluble.VolubleSourceConnector',
+
+  'genkp.people.with' = '#{Internet.uuid}',
+  'genv.people.name.with' = '#{Name.full_name}',
+  'genv.people.creditCardNumber.with' = '#{Finance.credit_card}',
+
+  'global.throttle.ms' = '500'
+);
+```
+
+## Modes
+
+Before you can use connector management, you need to decide what mode you want to run connectors in. ksqlDB can run connectors in two different modes: **embedded** or **external**. This controls how and where the connectors are executed. The way in which you configure ksqlDB's server determines which mode it will use. All nodes in a single ksqlDB cluster must use the same mode.
+
+Regardless of which mode you use, the syntax to create and use connectors is the same.
+
+## Embedded mode
+
+In embedded mode, ksqlDB runs connectors directly on its servers. This is convenient because it reduces the number of moving parts that you need to manage in your infrastructure. Embedded mode is highly useful for development, testing, and production workloads that have light/moderate data volumes. Use this mode when you don't need to scale your ingest/egress capacity independently from your processing capacity. When you use embedded mode, ksqlDB server is actually running a Kafka Connect server in distributed mode.
```

Comment:
```suggestion
In embedded mode, ksqlDB runs connectors directly on its servers. This is convenient because it reduces the number of moving parts that you need to manage in your infrastructure. Embedded mode is highly useful for development, testing, and production workloads that have light/moderate data volumes. Use this mode when you don't need to scale your ingest/egress capacity independently from your processing capacity. When you use embedded mode, ksqlDB server is actually running a {{ site.kconnect }} server in distributed mode.
```